metaci/release/models.py

```diff
@@ -13,6 +13,26 @@
 from metaci.release.utils import update_release_from_github
 
 
+class ReleaseCohort(models.Model):
+    name = models.CharField(_("name"), max_length=255)
+    status_choices = [
+        ("None", "None"),
+        ("Active", "Active"),
+        ("Cancelled", "Cancelled"),
+        ("Planned", "Planned"),
+    ]
+    status = models.CharField(
+        max_length=9,
+        choices=status_choices,
+        default="None",
```

Comment:
When would we use status="None"? Should we remove that option and default to Planned?